src/util.rs

```diff
@@ -51,6 +51,10 @@ pub struct Config {
     pub feature_peripheral: bool,
     #[cfg_attr(feature = "serde", serde(default))]
     pub max_cluster_size: bool,
+    #[cfg_attr(feature = "serde", serde(default))]
+    pub impl_register_debug: bool,
```

Comment:
I'd like it if this field took a enum instead, which would make it possible to use defmt instead in the future.

Ofcourse, this can be added later with a breaking change

Reply:
that or add another bool option for impl_defmt allowing both to be specified.